src/Classical/linear_code.jl

```diff
@@ -425,7 +426,30 @@ function random_information_set(C::AbstractLinearCode)
     return pivot_inds
 end
 
-
+#TODO doc
+function random_linear_code(field::CTFieldTypes, n::Int, k::Int, rng::AbstractRNG = Random.seed!())
+    rand_mat = zero_matrix(field, k, n-k)
+    for r in 1:nrows(rand_mat) 
+        for c in 1:ncols(rand_mat) 
+            rand_mat[r, c] = _rand_gf_elem(field, rng) 
+        end
+    end
+    full_mat = hcat(identity_matrix(field, k), rand_mat)
+    return LinearCode(full_mat)
+end 
+
+#TODO doc
```

Comment:
"""
random_linear_code(q::Int, n::Int, k::Int; rng::AbstractRNG = Random.seed!())

Return a random `[n, k]` linear code over `GF(q)`.
"""

indent, add a colon before the `rng` to make optional keyword parameter